pkgs/shells/fish/wrapper.nix

```diff
@@ -1,52 +1,109 @@
-{ lib, writeShellApplication, fish, writeTextFile }:
-
-lib.makeOverridable ({
-  completionDirs ? [],
-  functionDirs ? [],
-  confDirs ? [],
-  pluginPkgs ? [],
-  localConfig ? "",
-  shellAliases ? {},
-  runtimeInputs ? []
+{
+  lib,
+  writeShellApplication,
+  fish,
+  writeTextFile,
 }:
 
-let
-  aliasesStr = builtins.concatStringsSep "\n"
-      (lib.mapAttrsToList (k: v: "alias ${k} ${lib.escapeShellArg v}") shellAliases);
+lib.makeOverridable (
+  /**
+    Create a wrapped fish shell binary with some plugins, completions,
+    configuration snippets and functions sourced from the given paths.
+    This provides a convenient way to test Fish plugins and scripts
+    without having to alter the environment.
 
-  shellAliasesFishConfig = writeTextFile {
-    name = "wrapfish.aliases.fish";
-    destination = "/share/fish/vendor_conf.d/aliases.fish";
+    # Example
```

Comment:
I think you could use the fenced-divs as shown in the /docs/readme.md 

Like this:

:::{.example}

Example block

:::